models/Post.js

```diff
@@ -1,17 +1,19 @@
 var keystone = require('keystone');
 var transform = require('model-transform');
 var Types = keystone.Field.Types;
+var S3Storage = require('keystone-s3');
 
 var Post = new keystone.List('Post', {
 	autokey: { path: 'slug', from: 'name', unique: true },
 });
 
+
 Post.add({
 	name: { type: String, required: true },
 	state: { type: Types.Select, options: 'draft, published, archived', default: 'draft', index: true },
 	author: { type: Types.Relationship, ref: 'User', index: true },
 	publishedDate: { type: Types.Date, index: true, dependsOn: { state: 'published' } },
-	image: { type: Types.CloudinaryImage },
+	image: { type: Types.File, storage: 'S3Storage' },
```

Comment:
This is probably why keystone is failing to start - storage needs to be a storage instance, it's a string here